port/zephyr/Kconfig

```diff
@@ -63,6 +63,10 @@ config GOLIOTH_AUTH_CERT_MBEDTLS_DEPS
 	  This is a convenience option when using mbedTLS as a library to make
 	  (D)TLS connection with Golioth cloud.
 
+config MBEDTLS_MPI_MAX_SIZE
+	int
+	default 512 if GOLIOTH_AUTH_CERT_MBEDTLS_DEPS
+
```

Comment:
Would be good to include information about WHY we are doing it, i.e. information included in commit message in https://github.com/golioth/golioth-zephyr-sdk/commit/9f33611bd74380407008a2197671502e9842ca89